data/nlu/nlu.md

```diff
@@ -1,2878 +1,4910 @@
+- what's the weather today?
+- whats the temperature
+- whats the temperature in [delhi](location)?
+- whats the weather in [berlin](location)?
+- whats the weather like [tomorrow](DATE)?
+- wheather at you location?
+- wheather be like at your place?
 
 ## intent:ask_whatismyname
-- what is my name?
-- May i know my name?
-- What is my name?
-- What do I call myself?
-- Tell me my name.
-- Explain my name to me.
-- What could be my name?
-- What name do I go by?
-- What name should I recognize for myself?
-- what's my name?
-- can you tell me my name?
-- What is my name?
+- Can you tell me my name?
+- Can you tell me what I am called?
 - Do I have a name?
 - Do I have a name? What is it?
-- What was I named?
-- Say my name.
 - Do you know my name?
-- What do people call me?
-- Can you tell me my name?
-- What else do people call me?
+- Do you know what my name is?
+- Explain my name to me.
 - I have a name, what is it?
-- Tell me my name
 - I'd like to know my name
+- I'd like to know what my name is
 - Inform me what my name is
+- May i know my name?
 - Please let me know what my name is
-- Can you tell me what I am called?
-- I'd like to know what my name is
-- What is the name I was given?
-- What does everyone call me?
+- Say my name.
+- Tell me my name
+- Tell me my name.
 - What am I called?
+- What could be my name?
 - What did my parents name me?
+- What do I call myself?
 - What do my colleagues call me?
-- What would be the name on my tombstone?
-- Do you know what my name is?
-- What is my first name?
-- What is my full name?
 - What do people call me?
+- What does everyone call me?
+- What else do people call me?
 - What exactly is my name?
-
-
-## intent:ask_wherefrom
-- where do you live
-- where are you from?
-- So where are you from
-- Where are you?
-- Where were you born?
-- What city were you born in?
-- Where did you grow up?
-- What state were you born in?
-- What country were you born in?
-- Where do you come from?
-- Where were you at before you were here?
-- Where from?
-- Do you know where you come from?
-- where's your home town?
-- what country are you from?
-- are you from around here?
-- where did you grow up?
-- where are your parents from?
-- where did you spend your youth?
-- where did you come from?
-- Where were you born?
-- Where do you consider home?
-- What city do you claim to for your birth?
-- What citizenship do you lay claim to?
-- What is your heritage?
-- What is your birthplace?
-- Where did you grow up?
-- You were conceived in what location?
-- What is your root?
-- What is your origin?
-- From where did you come?
-- Where did you come from?
-- Where did you originate?
-- Where are your roots?
-- Where are your origins?
-- What area are you from?
-- What location are you from?
-- What's the name of the place you came from?
-
-## intent:ask_whoami
-- Who am I?
-- Okay who am I?
-- WHo am I
-- Tell me who am I?
-- Tell me who I am?
-- Do you know who I am?
-- Can you tell me who I am?
-- How 
```

Comment:
more duckling entities

Reply:
ugh these commented on the old and not the new i'm sorry, if you view the changed files in split mode you should see it reference the right thing